lib/galaxy/model/security.py

```diff
@@ -656,6 +656,18 @@ def associate_action_dataset_role(self, action, dataset, role):
         self.sa_session.flush()
         return assoc
 
+    def create_user_role(self, user, app):
+        # Create private user role if necessary
+        if not self.get_private_user_role(user):
+            self.create_private_user_role(user)
```

Comment:
I'd eliminate the if and just do ``self.get_private_user_role(user, auto_create=True)`` - no need to duplicate that if check logic and dispatching to create_private_user_role.

Reply:
Yea. Thank you. I saw that function, we use it only at one location with the true flag and it did not look very intuitive. I thought I'll rather refactor that one location and then get rid of the auto_create=true option but either way is fine. I'll revisit this in dev.